tutorials/03-astropy_dev_env.rst

```diff
@@ -0,0 +1,159 @@
+How to set up a development version of Astropy
+==============================================
+
+Found a bug? Know how to fix it? See a modification you need for your
+own project? This tutorial will teach you how to set up a conda
+environment for installing and modifying a developer version of Astropy.
+
+Pre-requisites
+--------------
+
+-  `Git <https://git-scm.com/>`__ version control system. On a Mac, this
+   will come with installing the latest version of the `Xcode
+   developer <https://developer.apple.com/xcode/>`__ tools.
+-  An account on `GitHub <https://github.com/>`__
+
+This tutorial will show you how to fork, clone, and install the
+development version of Astropy from the command line (terminal).
+
+Step 1: Fork the Astropy repo on Github
+---------------------------------------
+
+-  Log into your account on Github
+-  Go to https://github.com/astropy/astropy and click on **Fork** in the
+   upper right-hand corner of the page. This will create a separate repo
+   in your Github account under github.com//astropy
+
+Step 2: Clone your forked Astropy repo to your computer
+-------------------------------------------------------
+
+-  Go to your forked version of the repository and click on **Clone or
+   download** (the green button on the upper right-hand side of the
+   page).
+-  Highlight and copy the revealed URL. If you prefer using SSH to
+   connect to Github, you can toggle between "Use SSH" and "Use HTTPS"
+   by clicking on the highlighted text in the upper right-hand corner of
+   the box.
+-  Open a terminal session
+-  Make an astropy folder in the location you would like to develop the
+   code. For example:
+
+   ::
+
+       mkdir astropy-dev
+       cd astropy-dev
+       git clone git@github.com:<user-name>/astropy.git .
+
+**IMPORTANT NOTE:** If it has been a long time since you forked the
+Astropy repo for the first time, your fork may out of date. Before
+proceeding, follow the instructions for **Keeping your fork up to date**
+at the bottom of this page.
+
+Step 3: Create a new conda environment for developing astropy
+-------------------------------------------------------------
+
+-  Inside the astropy-dev folder, create a new conda environment called
+   *astropy-dev* and activate it.
+
+   ::
+
+       conda create -n astropy-dev python=3 anaconda
```

Comment:
I think we *don't* want anaconda here because that will include a conda install of astropy (which is included in the anaconda distribution). This is probably better:

```
conda create -n astropy-dev --only-deps python=3 astropy
```
That will create an environment with all of astropy's dependencies without installing astropy itself.

Reply:
Done, and added a note describing the only-deps flag